spec-main/fixtures/apps/set-path/main.js

```diff
@@ -1,8 +1,12 @@
 const http = require('http');
 const { app, ipcMain, BrowserWindow } = require('electron');
 
+let partition;
 if (process.argv.length > 3) {
   app.setPath(process.argv[2], process.argv[3]);
+  if (process.argv.length > 4) {
+    partition = 'persist:' + process.argv[4];
+  }
```

Comment:
We're getting to a point where maybe we want to consider arg parsing in this main.js